native-image-workshop/README.md

```diff
@@ -179,11 +196,11 @@ relies on reflection. A good candidate for testing this out would be to add `log
 We've already added it as a dependency in the `pom.xml` file, and it can be seen in the depencies:
 
 ```xml
-        <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
-        </dependency>
+<dependency>
+    <groupId>log4j</groupId>
+    <artifactId>log4j</artifactId>
+    <version>1.2.17</version>
+</dependency>
 ```
 
 To add `log4j` all we need to do is to open up the `ListDir.java` file and uncomment some things in order to start using it. Go through and uncomment the various lines that add the imports and the logging code. These are the following lines that need uncommenting:
```

Comment:
to open up  > to open the `ListDir.java` file and uncomment some things to start using it.

These are the following lines that need uncommenting: > Uncomment the following lines:

Reply:
updated